components/builder-jobsrv/src/server/scheduler.rs

```diff
@@ -745,3 +773,203 @@ fn buildable(project: &jobsrv::JobGroupProject) -> bool {
     matches!(project.get_state(),
              jobsrv::JobGroupProjectState::NotStarted | jobsrv::JobGroupProjectState::InProgress)
 }
+
+fn check_dispatchable(group: &jobsrv::JobGroup, name: &str) -> bool {
```

Comment:
was there a reason for moving this out of the struct?

Reply:
There is no dependence on self so I thought it may be more testable from outside as a free function.  But since I'm not using it in a test case it is not useful.  Will revert.